config/co2eq_parameters.js

```diff
@@ -6,10 +6,10 @@ var defaultCo2eqFootprint = {
     'gas': 490,
     'hydro': 24,
     'nuclear': 12,
-    'oil': 650,
+    'oil': 650, // UK Parliamentary Office of Science and Technology (2006) "Carbon footprint of electricity generation"
```

Comment:
This probably mean that we can't use "default source" (which is IPCC) for all default footprints.
I suggest we change the `defaultCo2eqFootprint` object from a hash giving the co2eq value to a hash giving the pair (co2eq, source). This will enable us to specify IPCC for all except oil.
This means we should change L53 to work accordingly.